connection.go

```diff
@@ -293,6 +300,13 @@ type SslOpts struct {
 	Ciphers string
 }
 
+func (opts Opts) Copy() Opts {
```

Comment:
The method Copy() has no description, but it is a public API.

Reply:
Added